components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/interceptors/auth/OAuthAuthenticationInterceptor.java

```diff
@@ -71,14 +73,21 @@ public void handleMessage(Message inMessage) {
         //set the request_authentication_scheme property in the message as oauth2.
         String accessToken = RestApiUtil.extractOAuthAccessTokenFromMessage(inMessage,
                 RestApiConstants.REGEX_BEARER_PATTERN, RestApiConstants.AUTH_HEADER_NAME);
+        String backendJWT = RestApiUtil.extractOAuthAccessTokenFromMessage(inMessage,
+                RestApiConstants.REGEX_BEARER_PATTERN, RestApiConstants.BACKEND_JWT_AUTH_HEADER_NAME);
```

Comment:
Should the backend JWT come in x-jwt-assertion: Bearer <token> format?
It is coming via just x-jwt-assertion: <token>